packages/core/src/components/ChoiceList/Choice.jsx

```diff
@@ -8,13 +8,6 @@ import uniqueId from 'lodash.uniqueid';
 /** Used to emit events to all Choice components */
 const dsChoiceEmitter = new EvEmitter();
 
-/**
```

Comment:
I didn't realize there was also documentation in the `jsx` files for the React section. I moved those into the relevant `docs.scss` pages as well